templates/emails/invitation-to-keynote-speaker.md

```diff
@@ -0,0 +1,11 @@
+## Template for Inviting folks to keynote
+
+Hello {{POTENIAL_KEYNOTE_SPEAKER_FIRST_NAME}},
+
+We hope you are in good health and doing well. This email is an invitation for you to be a keynote speaker for this year's PyCon India. PyCon India is the premier conference for the Python programing language in India and has been running for well over 12 years, scaling in both size and attendees. Last year, we had over 300+ proposals! What's better, we are completely non-profit and volunteer-run through and through, helping promote Python through many communities around India. 
+
+With the current situation [1], we, the conference organizers, are taking the unique opportunity to put more effort into making our conference bigger, more inclusive, and much more accessible to all our attendees. Working on this new direction, we would be glad to see you at our conference. We believe your experience and insights with Python and/or with Python community will be a critical part in making this conference a huge success! 
```

Comment:
Just used Grammarly on this

```suggestion
With the current situation [1], we, the conference organizers, are taking the unique opportunity to put more effort into making our conference bigger, more inclusive, and much more accessible to all our attendees. Working on this new direction, we would be glad to see you at our conference. We believe your experience and insights with Python and/or with Python community will be a critical part of making this conference a huge success! 
```

Reply:
Thanks @Bhupesh-V for the suggestion. I use Grammarly with a pinch of salt or sometimes not use it at all. Can you explain your suggestion on how using `of` instead of `in` for the preposition here would be better grammatically?